Dockerfile

```diff
@@ -87,6 +88,12 @@ COPY stack/clear-compressor-cache.py /rapidpro/
 EXPOSE 8000
 COPY stack/startup.sh /
 
+RUN mkdir -p /etc/supervisor/conf.d/
+RUN mkdir -p /var/log/supervisor
+
+COPY supervisor_rapidpro.conf /etc/supervisor/conf.d/rapidpro.conf
+COPY supervisord.conf /etc/supervisord.conf
```

Comment:
Prefer to do these things in as few steps as possible:
In this repo:
```
- supervisor/
  - supervisord.conf
  - conf.d/
    - rapidpro.conf
```

Then
```dockerfile
COPY supervisor /etc/supervisor
```
Don't need to make the conf directories then either.